Venmo/src/test/java/com/braintreepayments/api/VenmoClientUnitTest.java

```diff
@@ -772,7 +772,7 @@ public void tokenizeVenmoAccount_whenVenmoApiError_forwardsErrorToListener_andSe
         sut.tokenizeVenmoAccount(activity, request);
 
         verify(listener).onVenmoFailure(graphQLError);
-        verify(braintreeClient).sendAnalyticsEvent("pay-with-venmo.app-switch.failed");
+        verify(braintreeClient).sendAnalyticsEvent("pay-with-venmo.app-switch.failed", null);
```

Comment:
For these app switch methods, do we not get a `contextId` back at this point? I think we do on iOS just don't remember on android if we'd have one.

Reply:
I added couple of tests here [2d484a9](https://github.com/braintree/braintree_android/pull/934/commits/2d484a9c5ecc25813534f3e86d0ed84f84a74959)

For this particular test, we dont get a pyament context id at this point, since we are testing a GraphQl error https://github.com/braintree/braintree_android/blob/afe52117ef13a02fbf0bda6286a14fdcd0da2b6b/Venmo/src/test/java/com/braintreepayments/api/VenmoClientUnitTest.java#L760